controller/instance_handler.go

```diff
@@ -158,6 +159,10 @@ func (h *InstanceHandler) ReconcileInstanceState(obj interface{}, spec *types.In
 			logrus.Errorf("%v", err)
 			return err
 		}
+	} else if status.CurrentState == types.InstanceStateError && pod != nil {
+		if logs, err := h.getPodLogs(pod.Name, 100); err == nil {
```

Comment:
Make `100` a constant.

And don't ignore err even it's only for getting the log. Either log the error at the debug level, or explain in the comment that you have a good reason to ignore the error without side-effect.